### PR TITLE
Add a Game Manual

### PR DESCRIPTION
**Pull request by tannerwelsh**
Provide an outline of and instructions for the game.

### CODE REVIEW

**Comment by tannerwelsh** [comment]
@shereefb what do you think?

File: Game_Manual/Basic_Gameplay.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Every cycle, learners select learning goals, form projects from them, practice their craft in the context of the project, and finish the cycle by reflecting on both the process and the outcome.
+
+Learners complete cycles on their journey to achieving dignified livelihood.
+
+### Goal Selection
+
+The first stage of a cycle is called **goal selection**.
+
+During this stage, learners vote on goals from the goal library for their particular craft. The goal library for your craft is located in the appropriate craft repo under the [GuildCrafts][guildcrafts] organization.
+
```
Comment (shereefb) [inline Game_Manual/Basic_Gameplay.md:26]: Worth mentioning here that learners can also add to the goal library.

File: Game_Manual/Basic_Gameplay.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Each learner chooses their top two goals from the list. It's best to choose goals that are both _relevant_ to the skills you wish to acquire and appropriately _challenging_ given your current skill level.
+
+Use the `/vote` command to vote on goals.
+
+After all learners have voted, the votes are tallied and the most popular goals become **projects** to which learners are then assigned.
+
+### Project Kickoff
+
+A **project** is formed from a goal and is assigned a **team** of learners.
+
```
Comment (shereefb) [inline Game_Manual/Basic_Gameplay.md:46]: Might be worth diving a little deeper here into the goals of the team formation. Not everyone will get what they vote for. Team leads are automatically selected, projects are selected based on availability of team leads, votes, previous teams...etc.

File: Game_Manual/Basic_Gameplay.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+The purpose of a project is for the team to achieve the goal through practicing their craft, learning through collaboration and engagement with relevant and challenging tasks.
+
+#### Artifact and Project Outline
+
+Once projects have been formed, the first thing the team needs to do is to create a container for the **artifact** and write a **project outline** document within it.
+
+The artifact is the output of a project. It is the thing produced by a team to complete a project. All artifacts need a URL and a project outline document. In most cases, the artifact will be a GitHub repository in the account of a learner on the team.
+
+The project outline document defines the specifications and quality criteria for the artifact to meet. To create a project outline, copy the text of the goal into a `README.md` file within your artifact.
+
```
Comment (shereefb) [inline Game_Manual/Basic_Gameplay.md:56]: Worth adding: "Discuss the specs with your team, and adjust them as you see fit to get to a "stretch"

File: Game_Manual/Echo.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+Echo is under active development. If you notice something that could be better, or you encounter a bug, please log them in the [LOS repo][los-repo]. We want your feedback. Together we can build a more perfect Learning Operating System.
+
+## Commands
+
+Echo supports a variety of commands for interacting with the game.
+
+These include commands like `log` for logging reflections and notes, and `vote` for voting on goals.
+
+To issue a command, type a forward slash `/` followed by the command name in the chat window of Echo. You can type commands from any channel.
+
```
Comment (shereefb) [inline Game_Manual/Echo.md:14]: Is there a command that gets me a list of all echo commands?

Comment (tannerwelsh) [inline Game_Manual/Echo.md:14]: See next line:

```
When you type `/`, Echo will show a menu of the available commands and some hints as to their usage.
```

**Comment by tannerwelsh** [comment]
Thanks @shereefb! Updated based on feedback. Merging now...

**Comment by shereefb** [comment]
Wheeee.......

**Pull request merged**